### PR TITLE
Implemented batch commands

### PR DESCRIPTION
**Pull request by tboerger**
As I messed up https://github.com/crowbar/crowbar-client/pull/55 this continues it.

### CODE REVIEW

File: lib/crowbar/client/app/backup.rb [hunk 1]
```diff
@@ -130,7 +137,9 @@ def upload(file)
           "Delete a backup"
 
         long_desc <<-LONGDESC
-          `delete ID` will delete a backup from the Administration Server.
+          `delete ID` will delete a backup from the server. Be carefule
```
Comment (MaximilianMeister) [inline lib/crowbar/client/app/backup.rb]: s/carefule/careful

File: lib/crowbar/client/app/batch.rb [hunk 2]
```diff
@@ -28,24 +28,29 @@ class Batch < Base
           stdin you should just write a `-` instead of a specific
           filename.
 
-          With --include <barclamp[.proposal]> option you can process
-          only a specific part from the provided YAML file structure.
+          With --includes BARCLAMP[.PROPOSAL] option you can process
+          only a specific part from the provided YAML file structure,
+          if you don't provide a proposal name we will take the `default`
```
Comment (MaximilianMeister) [inline lib/crowbar/client/app/batch.rb]: who is we ? :)

**Comment by tboerger** [comment]
The fixes for backups by name will be fixed in followup pull requests.

**Comment by MaximilianMeister** [comment]
to me it looks good +1

(but nevertheless it would be good to test the batch on a running system)

**Comment by rsalevsky** [comment]
+1

**Pull request merged**